ui/app/controllers/vault/cluster/auth.js

```diff
@@ -81,8 +80,10 @@ export default Controller.extend({
       this.authSuccess(authResponse);
     },
     onMfaErrorDismiss() {
-      this.set('mfaAuthData', null);
-      this.auth.set('mfaErrors', null);
+      this.setProperties({
```

Comment:
I realize this is a non-glimmerized controller, but I'm curious if you have any thoughts on `setProperties`? When recently converting a component to glimmer, I needed to continue to use setProperties, but with the new glimmer format had to import it from ember/object. I see the API has it document in 4.x, but what's the benefit in this case of using it instead of doing something like:

this.mfaAuthData = null,
this.mfaErrors = null,

Reply:
Similar to `set`, `setProperties` will update properties on an object and then trigger a re-render in the case of a classic component. With glimmer components this pattern was replaced with tracked properties. For example, in a classic component:
```js
this.setProperties({ foo: true, bar: false });
```
In Glimmer would become:
```js
@tracked foo;
@tracked bar;
this.foo = true;
this.bar = false;
```
`setProperties` when imported from `@ember/object` was handy when using a POJO in a classic component where some of the properties were bound in the template.
```js
const foo = { bar: true };
setProperties(foo, { bar: false });
```
This would result in the template being re-rendered and the value of `bar` would be updated.

This is my understanding of it and as far as I can tell you shouldn't need `set` or `setProperties` in glimmer components but instead use tracked properties.

Reply:
Awesome, thank you for the info! When I finally get around to having that glimmerize component ready for review, I'll show you the use case in which I kept the setProperties. It had to do with a spread of properties.